Userland/Libraries/LibGfx/ImageFormats/JBIG2Loader.cpp

```diff
@@ -157,6 +157,35 @@ struct Code {
     u16 code {};                  // "Encoding" in spec.
 };
 
+// Table B.1 – Standard Huffman table A
+constexpr Array standard_huffman_table_A = {
+    Code { 1, 4, 0, 0b0 },
+    Code { 2, 8, 16, 0b10 },
+    Code { 3, 16, 272, 0b110 },
+    Code { 3, 32, 65808, 0b111 },
+};
+
+// Table B.2 – Standard Huffman table B
+constexpr Array standard_huffman_table_B = {
+    Code { 1, 0, 0, 0b0 },
+    Code { 2, 0, 1, 0b10 },
+    Code { 3, 0, 2, 0b110 },
+    Code { 4, 3, 3, 0b1110 },
+    Code { 5, 6, 11, 0b11110 },
+    Code { 6, 32, 75, 0b111110 },
+    Code { 6, 0, OptionalNone {}, 0b111111 },
```

Comment:
Would plain `{}` result in 0 or do you want to just be extra explicit here?

Reply:
I wanted to be extra explicit. Just `{}` should do the right thing, I think.